packages/perseus/src/__tests__/__snapshots__/perseus-markdown_test.jsx.snap

```diff
@@ -21,23 +21,14 @@ exports[`perseus markdown output should output columns 1`] = `
     <div
       className="perseus-column"
     >
-      <div
-        className="sat-header-grafting-area"
```

Comment:
I verified these element classnames are only used in the legacy SAT part of webapp and nowhere else.